tests/XmlTest.php

```diff
@@ -138,10 +146,8 @@ function testXml()
         $resp->addSpeak("Hello, welcome to the Jungle.", []);
       
         $output = $resp->toXML(true);
-     
-        self::assertXmlStringEqualsXmlString(
```

Comment:
@kunal-plivo why is this changed from assertXmlStringEqualsXmlString to assertEquals?

Reply:
Hey @nixonsam have made the change, sorry I do not recall why was this changed

have run the tests locally and everything is passing

phpunit --configuration tests/phpunit.xml
PHPUnit 7.4.3 by Sebastian Bergmann and contributors.

...............................................................  63 / 135 ( 46%)
............................................................... 126 / 135 ( 93%)
.........                                                       135 / 135 (100%)

Time: 205 ms, Memory: 14.00MB

have reverted the change as well